src/diagram/models/mathjs-utils.ts

```diff
@@ -29,19 +29,8 @@ export const getMathUnit = (value: number, unitString: string): math.Unit | unde
   }
 };
 
-// 1. use string replace of the inputNames with backticks
-// 2. then find all symbols and look for matches in those symbols
-// 3. if they match replace them 
 export const replaceInputNames = (expression: string, inputNames: (string | undefined)[]) => {
-  let localExpression = expression;
-
-  inputNames.forEach((name, index) => {
-    if (!name) {
-      return;
-    }
-    // A RegExp is used so all matches are replaced not just the first one
-    localExpression = localExpression.replace(RegExp("`" + name + "`", "g"), `input_${index}`);
-  });
```

Comment:
This was meant to support variable names with spaces and the use of backticks in equations to utilize such variable names. We're going to remove it since we no longer support variables with spaces in the names. Any old equations that use such variables will throw an error, but the user can just edit the variable names and remove the backticks to clear that it.